dendrite-config.yaml

```diff
@@ -0,0 +1,73 @@
+# The config file version format
+version: v0
+
+# The matrix specific config
+matrix:
+    # The name of the server. This is usually the domain name, e.g 'matrix.org', 'localhost'.
+    server_name: "example.com"
+    # The path to the PEM formatted matrix private key.
+    private_key: "/var/dendrite/matrix_key.pem"
```

Comment:
homedir? i would asume that dendrite is packaged as deb/rpm and it will use standard places for binaries, config files, certificates (`/etc/ssl/` or similar), whatnot. `/var/dendrite` is wrong.